### ISSUE TITLE
Cannot switch profile using the profile dropdown 

### ISSUE THREAD

**Issue by waldner** [IJHack/QtPass#687]
**Describe the bug**

Choosing a profile from the "select profile" dropdown menu does nothing. The currently displayed profile does not change.
To force a profile change, I have to open configuration -> profiles -> manually enter the desired profile path in the "current path" textbox -> ok.

If I run the app from the command line, this is shown upon startup:

```
qt.core.qmetaobject.connectslotsbyname: QMetaObject::connectSlotsByName: No matching signal for on_profileBox_currentIndexChanged(QString)
```

which might have something to do with the issue, although I'm not sure how to interpret that. But this is also reported here https://github.com/IJHack/QtPass/issues/581 and it's not clear to me whether it's the same problem.

**Desktop (please complete the following information):**
- OS/Distribution: Archlinux
- Version 1.4.0

**Comment by principis** [comment]
Fixed in #681.

**Comment by LandiLorenzo** [comment]
Hi, 
I just fall down into the same problem today, I'm a new user. I'm not sure, is this fixed? cause I have the same problem and I ended up build from source with qmake from qt5. 

thanks, cheers. 

Desktop (please complete the following information):

OS/Distribution: Fedora 40
Version 1.4.0

**Comment by principis** [comment]
> Hi, 
> I just fall down into the same problem today, I'm a new user. I'm not sure, is this fixed? cause I have the same problem and I ended up build from source with qmake from qt5. 
> 
> thanks, cheers. 
> 
> Desktop (please complete the following information):
> 
> OS/Distribution: Fedora 40
> Version 1.4.0

I backported the fixes in Fedora 41, but not Fedora 40. I can backport them to Fedora 40 if you wish, but I would rather encourage users to upgrade.

**Comment by LandiLorenzo** [comment]
thanks for let me know, It's ok for me to not use fixed version since I built with qt5. I'll upgrade to fedora 41 on the next month, and eventually I'll upgrade to the dnf version. Unfortunately I cannot afford fedora 41 since I have 1Mb network :).

**Comment by principis** [comment]
> Hi, I'm an Arch user and I have the same issue.
> 
> I've used pacman and aur
> 
> OS: EndeavourOS kernel: 6.12.1-arch1-1

It's a packaging issue. Raise this with the package maintainer.

This issue is fixed in https://github.com/IJHack/QtPass/pull/681.

**Comment by eworm-de** [comment]
The absence of a release is a packaging issue? I disagree. 😜
Packaging hundreds of packages makes it impossible to follow each upstream's development close enough.

I will apply this for Arch now, but a proper release every now and then is very welcome.

**Comment by waldner** [comment]
Just installed the new version in archlinux and the bug is fixed! Thanks!

**Comment by principis** [comment]
> The absence of a release is a packaging issue? I disagree. 😜 Packaging hundreds of packages makes it impossible to follow each upstream's development close enough.
> 
> I will apply this for Arch now, but a proper release every now and then is very welcome.

Agreed, I should've phrased it differently. Backporting patches is also not my favorite thing to do, and quite tedious 😅 .
